setup.cfg

```diff
@@ -11,3 +11,10 @@ convention = numpy
 [isort]
 lines_between_types = 1
 profile = black
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = pymc/_version.py
+versionfile_build = pymc/_version.py
+tag_prefix = rel-
```

Comment:
You might want to use `tag_prefix = v` to match the current format of PyMC's release versions.